src/vma/event/event_handler_manager.cpp

```diff
@@ -958,10 +964,10 @@ void* event_handler_manager::thread_loop()
 		if (ret == maxevents) {
 			// increase the events array
 			maxevents *= 2;
-			p_events = (struct epoll_event *)realloc((void *)p_events, maxevents);
+			p_events = (struct epoll_event *)realloc((void *)p_events, sizeof(struct epoll_event)*maxevents);
 			BULLSEYE_EXCLUDE_BLOCK_START
 			if (!p_events) {
-				evh_logpanic("failed to allocate memory") ;
+				evh_logpanic("%s : realloc failure", __func__) ;
```

Comment:
try not to use the _func_ in the statement
can the 'throw_vma_exception("")' help to replace it?